<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>logoin</class>
 <widget class="QWidget" name="logoin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>登录</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="title">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>60</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>70</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:rgba(0,0,0,0);border-color:rgba(0,0,0,255);color: rgba(0, 0, 0,255);border-style:none;border-width:1px;border-radius:0px;font:29px &quot;方正小标宋简体&quot;;font-style:normal;font-weight: normal;text-decoration:blink;</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;信息管理系统登录&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="count">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgba(0,0,0,0);border-color:rgba(0,0,0,255);color: rgba(0, 0, 0,255);border-style:none;border-width:1px;border-radius:0px;font:20px &quot;黑体&quot;;font-style:normal;font-weight: normal;text-decoration:blink;</string>
         </property>
         <property name="text">
          <string>用户名：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="styleSheet">
          <string notr="true">background-color:rgba(0,0,0,11);border-color:rgba(0,0,0,255);color: rgba(0, 0, 0,255);border-style:solid;border-width:1px;border-radius:7px;font:18px &quot;Times New Roman&quot;;</string>
         </property>
         <property name="placeholderText">
          <string>输入用户名</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="words">
         <property name="minimumSize">
          <size>
           <width>60</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:rgba(0,0,0,0);border-color:rgba(0,0,0,255);color: rgba(0, 0, 0,255);border-style:none;border-width:1px;border-radius:0px;font:20px &quot;黑体&quot;;font-style:normal;font-weight: normal;text-decoration:blink;</string>
         </property>
         <property name="text">
          <string>密  码：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_2">
         <property name="styleSheet">
          <string notr="true">background-color:rgba(0,0,0,11);border-color:rgba(0,0,0,255);color: rgba(0, 0, 0,255);border-style:solid;border-width:1px;border-radius:7px;font:18px &quot;Times New Roman&quot;;</string>
         </property>
         <property name="echoMode">
          <enum>QLineEdit::Password</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QPushButton" name="yes">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="quit">
         <property name="minimumSize">
          <size>
           <width>80</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>80</width>
           <height>30</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>退出</string>
         </property>
         <property name="shortcut">
          <string>Ctrl+R</string>
         </property>
         <property name="autoRepeatDelay">
          <number>300</number>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>quit</sender>
   <signal>clicked()</signal>
   <receiver>logoin</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>154</y>
    </hint>
    <hint type="destinationlabel">
     <x>305</x>
     <y>165</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
